src/main/scala/com/ubirch/webui/models/authentification/BearerAuthStrategy.scala

```diff
@@ -121,8 +121,14 @@ trait AuthenticationSupport extends ScentrySupport[(UserInfo, MemberType)] with
             case Left(_) => halt(Unauthorized("did not find device"))
             case Right(value) => action(userInfo._1, value)
           }
-        } else halt(Unauthorized("logged in as a user when only a device can be logged as"))
-      case None => halt(Unauthorized("Error while logging in"))
+        } else {
+          logger.warn("FAILED AUTH: User tried to logged in as device")
+          halt(Unauthorized("logged in as a user when only a device can be logged as"))
```

Comment:
"logged in as a user when only a device can be logged in"
Flüchtigkeitsfehler? ;)